ci.yml

```diff
@@ -0,0 +1,42 @@
+# Configure which branches trigger builds
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+
+variables:
+- name: TeamName
+  value: AspNetCore
+
+resources:
+  repositories:
+  # Repo: 1ESPipelineTemplates/1ESPipelineTemplates
+  - repository: 1esPipelines
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    sdl:
+        policheck:
```

Comment:
Indented 1 too many times

Reply:
I'll run a formatter through this file and commit the changes.